src/muse/CMakeLists.txt

```diff
@@ -357,6 +357,10 @@ if(LV2_SUPPORT)
             OUTPUT_NAME muse_lv2host_module
             )
             
+      if(NOT HAVE_CXX_ATOMICS_WITHOUT_LIB)
```

Comment:
Hi, thank you for the patch. I am ready to merge, but I am curious...
Is atomic only required for our LV2 support module?
If so, would it be possible (and wise?) to move the all of additions in the top cmake file
  down a bit further and place them inside the 'if (ENABLE_LV2)' block near line 505?
Our LV2 support is optional, that is why I am asking.

Maybe something like this?
Line 505: if (ENABLE_LV2)
find_package(ECM REQUIRED NO_MODULE)
set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH})
include(CheckAtomic)

set (muse_sord_ver "0.16.0")
...

Reply:
Hi, I successfully tested your suggested changes and updated the commits. Sounds like a good idea. Thank you!